typescript/agentkit/src/wallet-providers/privyEvmWalletProvider.ts

```diff
@@ -0,0 +1,172 @@
+import { PrivyClient } from "@privy-io/server-auth";
```

Comment:
This file is almost an exact copy-paste of the pre-existing `privyWalletProvider.ts`. Any changes made are commented below